drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -542,18 +554,18 @@ int stm32_clock_control_init(const struct device *dev)
 
 	/* Enable HSE if not enabled */
 	if (LL_RCC_HSE_IsReady() != 1) {
-		/* Check if need to enable HSE bypass feature or not */
-		if (IS_ENABLED(STM32_HSE_BYPASS)) {
 #ifdef CONFIG_SOC_SERIES_STM32WLX
+		if (IS_ENABLED(STM32_HSE_TCXO)) {
 			LL_RCC_HSE_EnableTcxo();
-		} else {
-			LL_RCC_HSE_DisableTcxo();
+		}
```

Comment:
`else { LL_RCC_HSE_DisableTcxo(); }   ` ?  ... similar to Bypass

Reply:
Default configuration is TCXO disabled.
Then, the if condition is defined at build time, so there is no way we can enable then disable.

I think that the one to be fixed would be the bypass part.